src/components/EditableHeading/EditableHeading.module.scss

```diff
@@ -0,0 +1,36 @@
+@import "../../styles/typography";
+@import "~monday-ui-style/dist/mixins";
+
+@include create-title-classes;
```

Comment:
what is this exactly?

Reply:
This creates the new typography classes

Reply:
Ok, I see. It’s for the input element. Just add a comment there to make it more clear as it looks like a side affect that no one uses. Maybe even using it inside some class that you use to explain the connecting